packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -28,6 +28,18 @@ export const PRODUCT_JETPACK_VIDEOPRESS_MONTHLY = 'jetpack_videopress_monthly';
 export const PRODUCT_JETPACK_SOCIAL_BASIC = 'jetpack_social_basic_yearly';
 export const PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY = 'jetpack_social_basic_monthly';
 
+//add-on products
+export const PRODUCT_JETPACK_BACKUP_ADDON_10GB_MONTHLY =
+	'jetpack_backup_addon_storage_10gb_monthly';
+export const PRODUCT_JETPACK_BACKUP_ADDON_100GB_MONTHLY =
+	'jetpack_backup_addon_storage_100gb_monthly';
+export const PRODUCT_JETPACK_BACKUP_ADDON_1TB_MONTHLY = 'jetpack_backup_addon_storage_1tb_monthly';
+
+export const PRODUCT_JETPACK_BACKUP_ADDON_10GB_YEARLY = 'jetpack_backup_addon_storage_10gb_yearly';
```

Comment:
I thought I would also add `yearly` products, even though we are planning to launch only `monthly` for first iteration